codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/TypesGenerator.java

```diff
@@ -267,6 +293,19 @@ private void renderUnionToSMethod(MemberShape memberShape) {
         }
     }
 
+    private class MemberDefaultVisitor extends ShapeVisitor.Default<String> {
+
+        @Override
+        protected String getDefault(Shape shape) {
```

Comment:
What about for Strings, blobs, etc? Is default the right visitor method?

Reply:
Currently yes - only the number types and String are supported.  We could implement all of the numbers individually here or just use the default.  Thoughts?

Reply:
What's safest? If Blob is supported, then we're writing 0, right?